ci/do_ci.sh

```diff
@@ -171,6 +171,21 @@ elif [[ "$1" == "bazel.coverage" ]]; then
   # directory. Wow.
   cd "${ENVOY_BUILD_DIR}"
   SRCDIR="${GCOVR_DIR}" "${ENVOY_SRCDIR}"/test/run_envoy_bazel_coverage.sh
+  exit 0
+elif [[ "$1" == "bazel.clang_tidy" ]]; then
```

Comment:
Can you document this the other places in which we document the various CI build options for developers?

Reply:
Done